dist/api_data.py

```diff
@@ -1379,6 +1389,29 @@ def __ge__(self, other):
         for a transaction. See @ref transaction_timestamps'''),
 ]),
 
+'WT_SESSION.timestamp_transaction_numeric' : Method([
```

Comment:
@sueloverso do you have a naming preference for these two new APIs? We need to distinguish them from their equivalents that parse the number out of the config string.

Reply:
I cannot say I love the naming. But I don't have anything better to suggest either.

Reply:
After thinking about this a bit, what about `timestamp_transaction_value`. Although they're not the same thing it kind of flows with `cursor->set_value` and makes sense. I like that word a bit better than `_numeric`.